<a href="https://colab.research.google.com/github/farisassallami/Food-Sales-Predictions/blob/main/project2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Name: Faris Assallami

In [3]:
from google.colab import drive
drive.mount('/content/drive')
import pandas as pd

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [4]:
df1 = pd.read_csv('/content/Metabolic  Syndrome.csv')
df2 = pd.read_csv('/content/adult 2.csv')

#Data set 1 : Metabolic Syndrome

In [9]:
print(df1.shape)
df1.head()


(2401, 15)


seqn  Age     Sex  Marital  Income   Race  WaistCirc   BMI  Albuminuria  \
0  62161   22    Male   Single  8200.0  White       81.0  23.3            0   
1  62164   44  Female  Married  4500.0  White       80.1  23.2            0   
2  62169   21    Male   Single   800.0  Asian       69.6  20.1            0   
3  62172   43  Female   Single  2000.0  Black      120.4  33.3            0   
4  62177   51    Male  Married     NaN  Asian       81.1  20.1            0   

   UrAlbCr  UricAcid  BloodGlucose  HDL  Triglycerides MetabolicSyndrome  
0     3.88       4.9            92   41             84         No MetSyn  
1     8.55       4.5            82   28             56         No MetSyn  
2     5.07       5.4           107   43             78         No MetSyn  
3     5.22       5.0           104   73            141         No MetSyn  
4     8.13       5.0            95   43            126         No MetSyn

1. Source of data
*- The dataset for analysis came from the NHANES initiative*


2. Brief description of data
*- The following description of the data is divided into features of body vitals measurements abnormal waist circumference, triglycerides above 150, HDL cholesterol below 50 in women or 40 in men, history of hypertension and mildly elevated fasting blood sugar (100-125). Numerous other variables were added, such as uric acid, race, income, etc. that might contribute to the model but we will not sure, until we test the model.*

3. What is the target?
*-The target is Metabolic Syndrome*


4. What does one row represent? (A person?  A business?  An event? A product?)
*-One row represents a patient*


5. Is this a classification or regression problem?
*-This is a classification problem*


6. How many features does the data have?
*-There are 15 features*

7. How many rows are in the dataset?
*-There are 2401 rows*

8. What, if any, challenges do you foresee in cleaning, exploring, or modeling this dataset?
*-Significant amount of missing data*

#Data set 2 : Adult Income

In [11]:
print(df2.shape)
df2.head()

(48842, 15)


age  workclass  fnlwgt     education  educational-num      marital-status  \
0   25    Private  226802          11th                7       Never-married   
1   38    Private   89814       HS-grad                9  Married-civ-spouse   
2   28  Local-gov  336951    Assoc-acdm               12  Married-civ-spouse   
3   44    Private  160323  Some-college               10  Married-civ-spouse   
4   18          ?  103497  Some-college               10       Never-married   

          occupation relationship   race  gender  capital-gain  capital-loss  \
0  Machine-op-inspct    Own-child  Black    Male             0             0   
1    Farming-fishing      Husband  White    Male             0             0   
2    Protective-serv      Husband  White    Male             0             0   
3  Machine-op-inspct      Husband  Black    Male          7688             0   
4                  ?    Own-child  White  Female             0             0   

   hours-per-week native-country income  
0              40  United-States  <=50K  
1              50  United-States  <=50K  
2              40  United-States   >50K  
3              40  United-States   >50K  
4              30  United-States  <=50K

1. Source of data
*-US Census Bureau*

2. Brief description of data
*-A persons age, hours worked per week, income and occupation*

3. What is the target?
*-Income*

4. What does one row represent? (A person?  A business?  An event? A product?)
*-A person*

5. Is this a classification or regression problem?
*-Regression problem*

6. How many features does the data have?
*-There are 15 features*

7. How many rows are in the dataset?
*-There are 48842 rows*

8. What, if any, challenges do you foresee in cleaning, exploring, or modeling this dataset?
*-Missing significant amount of data*